### PR TITLE
Kubeflow has moved from google/kubeflow to its own GitHub organization.

### PR DESCRIPTION
**Pull request by jlewi**
* Update all references to google/kubeflow.
* Configure kubeflow as a trusted organization so that members of Kubeflow
  can trigger tests.

* Related to: kubeflow/kubeflow#165

### CODE REVIEW

File: prow/plugins.yaml [hunk 1]
```diff
@@ -18,7 +17,10 @@ triggers:
   - containerd/cri-containerd
   trusted_org: containerd
   join_org_url: "https://github.com/containerd/containerd/blob/master/MAINTAINERS"
-
+- repos:
+  - kubeflow/kubeflow
```
Comment (cjwagner) [inline prow/plugins.yaml]: Just use `kubeflow` to make this apply to the entire `kubeflow` Github org.

**Comment by krzyzacy** [comment]
assume you need to invite the bot again?

**Comment by jlewi** [comment]
@krzyzacy Yes. Thanks for the reminder. I'm going to setup webhooks and bot permissions for the new org.

File: prow/plugins.yaml (hunk 1 above)
Comment (jlewi) [reply prow/plugins.yaml]: Fantastic.

**Comment by jlewi** [comment]
Do I need to do something to fix the tide check?

**Comment by krzyzacy** [comment]
/lgtm
(meant to stamp earlier)

**Pull request merged**